src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_executable_network/get_metric.cpp

```diff
@@ -34,6 +34,15 @@ INSTANTIATE_TEST_SUITE_P(nightly_OVClassExecutableNetworkGetMetricTest,
                          OVClassExecutableNetworkGetMetricTest_ThrowsUnsupported,
                          ::testing::Values("GPU", "MULTI:GPU", "HETERO:GPU", "AUTO:GPU,CPU", "BATCH:GPU"));
 
+const std::vector<std::tuple<std::string, std::string>> GetMetricTest_ExecutionDevice_GPU = {
+        {"GPU", "GPU.0"},
+        {"GPU.0", "GPU.0"},
+        {"BATCH:GPU", "GPU.0"}};
```

Comment:
Could we add one more case for AUTO throughput mode when "ALL_AUTO_BATCHING" is "YES"?

Reply:
Added in 'src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp', thanks.